src/DebuggerForm.cpp

```diff
@@ -498,10 +498,10 @@ void DebuggerForm::createToolbars()
 void DebuggerForm::updateCustomActions()
 {
 	userToolbar->clear();
-	for (auto control: controls) {
-	        auto* action = new QAction(control.name, this);
-	        action->setStatusTip(control.description);
-		action->setIcon(QIcon(control.icon.isEmpty() ? ":/icons/gear.png" : control.icon));
+	for (auto command: commands) {
```

Comment:
`const auto&`
Maybe check all the range-for-loops. I won't repeat this review comment anymore.